router/router.nginx.conf.in

```diff
@@ -232,3 +232,19 @@ server {
     listen 80;
     listen [::]:80;
 }
+
+server {
+    server_name blog.*;
+
+    location / {
+        proxy_pass http://blog/;
+
+        proxy_set_header Host $updated_host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    }
+
+    listen 80;
+    listen [::]:80;
+	client_max_body_size 8M;
```

Comment:
Why did you add this?  It seems quite large.

Reply:
This would be useful for image/video uploads; we should set this to whatever the largest file size we want for media.

Reply:
I don't think this will make it past the gateway though

Reply:
I ran into this when trying to upload a 1.2MB theme. Adding this line worked for the dev deployed version; would gateway do something else for blog.hail.is?

And yeah, I set this semi-arbitrarily with an image/video upload limit in mind. Discord does ~7MB per post and I thought that was pretty reasonable.

Reply:
Gateway has this for internal URLs, which is honestly a little unsettlingly large?
```
server {
    server_name internal.hail.is;
    client_max_body_size 10000m;
```
You'll need a similar thing under:
```
server {
    server_name hail.is;
```
but maybe do 8M instead of 10000m?

Reply:
We wouldn't run into this for a while, but users using the eventual service may serialize IR's that are quite large right? We could enable larger body sizes for just the IR routes (to say /api/ with method POST)

Reply:
I think the original reason this was set was to allow for very large batches to be submitted in a single request, which we moved away from, so I agree that 10GB is much more than we need, at least until we have a specific use case (uploading could be one, if we ever allow users to submit files to the cluster, rather than S3/GCS).

Reply:
I assume HTTP body limits have a good reason to exist. The client of the Hail service is not a web browser, so it can break it's messages into arbitrarily small chunks.